manifests/f/FerroSoftware/FtpUse/2.2/FerroSoftware.FtpUse.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: FerroSoftware.FtpUse
+PackageVersion: 2.2
+PackageLocale: en-US
+Publisher: Ferro Software
+PackageName: FtpUse
+License: Copyright © 2000-2022 FERRO Software

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate